### ISSUE TITLE
lantern blocks have no waterlogged function

### ISSUE THREAD

**Issue by Alonlystalker** [CookieCNetwork/Chipped#57]
The game crashed whilst unexpected error
Error: java.lang.IllegalArgumentException: Cannot get property BooleanProperty{name=waterlogged, clazz=class java.lang.Boolean, values=[true, false]} as it does not exist in Block{chipped:special_lantern_3}
Код завершения: -1

**Comment by terrariumearth** [comment]
is this for the fabric or the forge version of the mod?

**Comment by Alonlystalker** [comment]
Forge

**Comment by CodexAdrian** [comment]
Fixed in latest version

**Comment by KMorris29** [comment]
I think this is still an issue in Chipped Forge  v1.1.2, although it expresses itself differently. When loading structure files with special_lantern_3's through a structure block, the game does not crash, but it does cause other blocks (especially modded blocks and/or tile entities like chests), to load in inconsistently. Removing all special lanterns and re-saving the structure allows the entire structure to be loaded correctly.
When looking at the logs for the moment of structure loading, it says something like this:
[22:10:38] [Render thread/INFO]: [CHAT] Structure 'minecraft:actonehallway' position prepared
[22:10:40] [Server thread/FATAL]: Error executing task on Server
java.lang.IllegalArgumentException: Cannot get property BooleanProperty{name=waterlogged, clazz=class java.lang.Boolean, values=[true, false]} as it does not exist in Block{chipped:special_soul_lantern_3}
at net.minecraft.state.StateHolder.func_177229_b(SourceFile:95) ~[?:?]
	at net.minecraft.block.IWaterLoggable.func_204509_a(SourceFile:20) ~[?:?]
	at net.minecraft.world.gen.feature.template.Template.func_237146_a_(Template.java:258) ~[?:?]
       (etc.)
while re-saving and loading the exact same structure without the lanterns just gives me this:
[22:11:43] [Render thread/INFO]: [CHAT] Structure 'minecraft:actonehallway' position prepared
[22:11:45] [Render thread/INFO]: [CHAT] Structure loaded from 'minecraft:actonehallway'

Simplest way to reproduce that I've found is to save a 1x2x1 structure of 2 lantern_3's. If loaded in an empty area, only the west or north lantern will appear, and you get the error. Loading the structure a second time from the exact same block, oddly enough, places the second lantern, but still gives you the error. Loading once for each time there is a lantern in the structure is a temporary fix that I have found.
